packages/safe-tools-client/app/components/create-safe-button/index.gts

```diff
@@ -36,18 +37,20 @@ export default class CreateSafeButton extends Component<Signature> {
     this.isModalOpen = true;
 
     try {
-      const [gasEstimate, nativeTokenBalance] = await Promise.all([
+      const [gasEstimate, nativeTokenBalance, currentGasPrice] = await Promise.all([
         this.scheduledPaymentsSdk.getCreateSafeGasEstimation(),
         this.wallet.fetchNativeTokenBalance(),
+        getCurrentGasPrice(this.wallet.network.chainId)
       ]);
 
+      const gasPrice = BigNumber.from(currentGasPrice?.standard.toString());
+      const gasEstimateInWei = gasEstimate?.mul(gasPrice);
```

Comment:
Is there any reason to not return calculated gasPrice estimation from the sdk directly ? I mean, instead of using  `getCurrentGasPrice` and multiplying here, could the `createSafeWithModuleAndGuardEstimation` already return the correct value in wei, so we just match it against the balance ? That being the case, no change on the frontEnd is needed

Reply:
You're right, updated